docs/getting-started.md

```diff
@@ -287,6 +287,16 @@ This will build and place the two Firecracker binaries at:
 - `build/cargo_target/${toolchain}/debug/firecracker` and
 - `build/cargo_target/${toolchain}/debug/jailer`.
 
+If you would like to test a new feature and work with
+dependencies on libraries located in private git repos, you
```

Comment:
This only adds support for a single set of credentials. Multiple private repos with different credentials is not supported.

I think this should be fine for now, but it is worth explicitly mentioning.

Reply:
Done.